eng/pipelines/libraries/stress/http.yml

```diff
@@ -0,0 +1,73 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+    - "*"
```

Comment:
this still looks weird to me. We discussed this behavior is implicit, right?

Reply:
Yeah, I guess I just wanted to make it more explicit, for self-documenting reasons.